### PR TITLE
Come up with a better format to indicate that a test requires the WorldBuilder.

### PR DESCRIPTION
**Pull request by bangerth**
I verified that it enables the same number of tests as before (990 on my machine).

/rebuild

### CODE REVIEW

**Comment by bangerth** [comment]
I pushed a second commit that does a similar check for the `QUICK_TEST` string.

File: tests/world_builder_select_composition.prm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# WORLD BUILDER
+# Requires: ASPECT_WITH_WORLD_BUILDER
```
Comment (tjhei) [inline tests/world_builder_select_composition.prm:1]: "Enable if" not "Requires"?

File: tests/world_builder_simple.prm [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# WORLD BUILDER
+# Requires: ASPECT_WITH_WORLD_BUILDER
```
Comment (tjhei) [inline tests/world_builder_simple.prm:1]: same here

File: tests/world_builder_simple_2d_cartesian_sticky_air.prm [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# WORLD BUILDER
+# Requires: ASPECT_WITH_WORLD_BUILDER
```
Comment (tjhei) [inline tests/world_builder_simple_2d_cartesian_sticky_air.prm:1]: and here

**Review by tjhei — COMMENTED** [review]
Did you forget to change the tests/*prm that contain ``QUICK_TEST``?

**Comment by bangerth** [comment]
Yes, I had those commits but somehow messes up the push... Let's see whether this works now!

File: tests/CMakeLists.txt [hunk 4]
```diff
@@ -97,20 +97,21 @@ ENDFUNCTION()
...
 # checking for the following strings in the .prm:
-# 1. "WORLD BUILDER" - only run with World Builder.
-# 2. "QUICK_TEST" - enable the test even if RUN_ALL_TESTS is false
+# 1. "Enable if: ASPECT_WITH_WORLD_BUILDER" - only run with World Builder.
+# 2. "Enable if: QUICK_TEST" - enable the test even if RUN_ALL_TESTS is false
 FUNCTION(SHOULD_ENABLE_TEST _filename)
 
   FILE(STRINGS ${_filename} _input_lines
-  REGEX "WORLD BUILDER")
+       REGEX "Enable if: +ASPECT_WITH_WORLD_BUILDER")
```
Comment (tjhei) [inline tests/CMakeLists.txt:105]:
```suggestion
       REGEX "Enable if: ASPECT_WITH_WORLD_BUILDER")
```
or add the + for quick tests as well to be consistent.

**Review by tjhei — APPROVED** [review]

**Pull request merged**